composer.json

```diff
@@ -96,6 +96,7 @@
         "doctrine/mongodb-odm-bundle": "To support MongoDB. Only versions 4.0 and later are supported.",
         "elasticsearch/elasticsearch": "To support Elasticsearch.",
         "guzzlehttp/guzzle": "To use the HTTP cache invalidation system.",
+        "ocramius/package-versions": "To display the API Platform's version in the debug bar.",
```

Comment:
@alanpoulain wouldn't it be more interesting to suggest https://github.com/composer/package-versions-deprecated for composer 1 & 2 compatibility?

Reply:
Not sure about this. If the user doesn't use PHP 7.4 or Composer 2, it will find this package anyway, don't you think?
And `ocramius/package-versions` is the original package, it makes sense to me to add it to the suggest list.

Reply:
Indeed 👍